kubernetes.yaml

```diff
@@ -181,7 +183,7 @@ rules:
     verbs: ["get"]
   - apiGroups: ["batch"]
     resources: ["jobs"]
-    verbs: ["get", "list", "create", "delete"]
+    verbs: ["get", "list", "create", "patch", "update", "delete"]
```

Comment:
do you need `patch` _and_ `update`? what is the difference?

Reply:
Yes, good point, since we do not update the entire resource, patch is enough for the feature, deleted it